int8 speedL
int8 speedR
